spec/models/company_value_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CompanyValue, type: :model do
+  it { is_expected.to validate_presence_of(:title) }
+  # it { shoulda validate_presence_of(:title) }
```

Comment:
we shouldn't leave commented unnecessary lines of code

Reply:
I removed it